docs/source/raft_ann_benchmarks.md

```diff
@@ -181,7 +182,7 @@ options:
 `result-filepaths` : whitespace delimited list of result files/directories that can be captured via pattern match. For more [information and examples](ann_benchmarks_low_level.html#result-filepath-example)
 
 #### Step 4: Plot Results
-The script `scripts/ann-benchmarks/plot.py` will plot all results evaluated to a CSV file for a given dataset.
+The script `bench/ann/plot.py` will plot all results evaluated to a CSV file for a given dataset.
```

Comment:
I think we need to add `bench/ann` to the README.md as well.

Reply:
Where to add it and in what capacity?

Reply:
Bottom of readme.md where we list all the paths in the codebase and what they are for.

Reply:
Done